.sass-lint.yml

```diff
@@ -0,0 +1,17 @@
+rules:
+  property-sort-order:
+    - 2
+    - order: 'smacss'
+  force-element-nesting:
+    - 0
+  no-color-literals:
+    - 2
+    -
+      allow-rgba: true
+  no-url-protocols:
+    - 0
+  nesting-depth:
+    - 2
+    -
+      max-depth: 3
+  no-vendor-prefixes: 0
```

Comment:
Should we set everything to warnings (1) instead of errors (2) ?
If we add sass linting as a webpack build step, it probably never should break for sass errors. What do you think?